api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -154,26 +153,28 @@ def aspirate(
         rate: float = 1.0,
     ) -> InstrumentContext:
         """
-        Aspirate a given volume of liquid from the specified location, using
-        this pipette.
+        Draw liquid into a pipette tip. Includes parameters that control
+        aspiration volume, well location, tip position in a well,and pipette flow rate.
+        See :ref:`new-aspirate`.
```

Comment:
Summarizing the immediately following parameter list is redundant. And let's keep the first line special.
```suggestion
        Draw liquid into a pipette tip.
        
        See :ref:`new-aspirate` for more details and examples.
```